.github/actions/list-variants/action.yml

```diff
@@ -8,17 +16,6 @@ outputs:
     description: Variants that should not run for aarch64
     value: ${{ steps.get-variants.outputs.aarch-enemies }}
 runs:
-  using: "composite"
-  steps:
-    - uses: actions/checkout@v3
-    - id: get-variants
-      name: Determine variants
-      shell: bash
-      run: |
-        cd variants
-        output="variants=$(ls -d */ | cut -d'/' -f 1 | grep -vE '^(shared|target)$' | jq -R -s -c 'split("\n")[:-1]')"
-        echo $output
-        echo $output >> $GITHUB_OUTPUT
-        output="aarch-enemies=$(ls -d */ | cut -d'/' -f 1 | grep -E '(^(metal|vmware)|\-dev$)' | jq -R -s -c 'split("\n")[:-1] | [ .[] | {"variant": ., "arch": "aarch64"}]')"
-        echo $output
-        echo $output >> $GITHUB_OUTPUT
+  using: "node20"
+  main: 'dist/index.js'
```

Comment:
I'm quite uncomfortable taking this dependency.

Reply:
This is the configuration setting for the action run to tell it to use node20. We are not actually importing any code into the project itself.

Is that what you are referring to?